spec/forms/registration_number_forms_spec.rb

```diff
@@ -4,40 +4,44 @@
   describe "#submit" do
     context "when the form is valid" do
       let(:registration_number_form) { build(:registration_number_form, :has_required_data) }
-      let(:valid_params) { { reg_identifier: registration_number_form.reg_identifier } }
+      let(:valid_params) { { reg_identifier: registration_number_form.reg_identifier, company_no: "09360070" } }
 
       it "should submit" do
-        expect(registration_number_form.submit(valid_params)).to eq(true)
+        VCR.use_cassette("registration_number_form_valid_company_no", re_record_interval: 14.days) do
+          expect(registration_number_form.submit(valid_params)).to eq(true)
+        end
       end
     end
 
     context "when the form is not valid" do
       let(:registration_number_form) { build(:registration_number_form, :has_required_data) }
-      let(:invalid_params) { { reg_identifier: "foo" } }
+      let(:invalid_params) { { reg_identifier: "foo", company_no: "foo" } }
 
       it "should not submit" do
         expect(registration_number_form.submit(invalid_params)).to eq(false)
       end
     end
   end
 
-  describe "#reg_identifier" do
-    context "when a valid transient registration exists" do
-      let(:transient_registration) do
-        create(:transient_registration,
-               :has_required_data,
-               workflow_state: "registration_number_form")
-      end
-      # Don't use FactoryBot for this as we need to make sure it initializes with a specific object
-      let(:registration_number_form) { RegistrationNumberForm.new(transient_registration) }
+  context "when a valid transient registration exists" do
+    let(:transient_registration) do
+      create(:transient_registration,
+             :has_required_data,
+             workflow_state: "registration_number_form")
+    end
+    # Don't use FactoryBot for this as we need to make sure it initializes with a specific object
+    let(:registration_number_form) { RegistrationNumberForm.new(transient_registration) }
 
+    describe "#reg_identifier" do
       context "when a reg_identifier meets the requirements" do
         before(:each) do
           registration_number_form.reg_identifier = transient_registration.reg_identifier
         end
 
         it "is valid" do
-          expect(registration_number_form).to be_valid
+          VCR.use_cassette("registration_number_form_valid_company_no", re_record_interval: 14.days) do
```

Comment:
VCR supposedly has the ability to apply [default cassette options](https://relishapp.com/vcr/vcr/v/3-0-3/docs/configuration/default-cassette-options) at the config level.

Rather than having to specify it in every call, I'd like to see if we can try and set it at the global level.

Reply:
Sounds good to me – I must have ended up on the wrong page of the docs. Will add this in!